api/src/main/java/org/openmrs/module/pihmalawi/sql/MysqlScriptRunner.java

```diff
@@ -0,0 +1,207 @@
+package org.openmrs.module.pihmalawi.sql;
+
+/*
+ * Slightly modified version of ScriptRunner
+ * https://github.com/BenoitDuffez/ScriptRunner
+ */
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.LineNumberReader;
+import java.io.PrintWriter;
+import java.io.Reader;
+import java.io.UnsupportedEncodingException;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class MysqlScriptRunner {
+
+    private static Log log = LogFactory.getLog(MysqlScriptRunner.class);
+
+    private static final String DEFAULT_DELIMITER = ";";
```

Comment:
I left on the top of the file the link from which this code came from. Here is the license: https://github.com/BenoitDuffez/ScriptRunner/blob/master/LICENSE